platform/ios/docs/guides/For Style Authors.md

```diff
@@ -17,7 +17,9 @@ When designing your style, consider the context in which your application shows
 the style. There are a number of considerations specific to iOS that may
 not be obvious when designing your style in Mapbox Studio on the Web. A map view
 is essentially a graphical user interface element, so many of same issues in
-user interface design also apply when designing a map style.
+user interface design also apply when designing a map style. For more
+information about designing maps for mobile, see [Nathaniel Slaughter's blog post](https://www.mapbox.com/blog/designing-maps-for-mobile-devices/) on
+the subject.
```

Comment:
This paragraph introduces the subsections below. This guide and the blog post cover somewhat different aspects of mobile map design, so I don’t want to downplay it. However, it might improve the flow if we move this sentence down to the section’s last paragraph, where we already link to the HIG.